ashpy/trainers/gan.py

```diff
@@ -78,7 +78,7 @@ class AdversarialTrainer(BaseTrainer):
                         filters_cap=32,
                         output_shape=10,
                     ),
-                    #model_selection_operator=operator.gt,
+                    model_selection_operator=operator.gt,
```

Comment:
This line was commented because model selection made the tests fail. How is it possible that now they're working?
Model selection produces something on the standard output that pytest captures, making the test to fail.

If the tests are passing even with this line present (not commented) then there should be something wrong

Reply:
Actually metrics use tf.print, so its output goes to stderr. So uncommenting the model selection operator is ok